contribs/gmf/less/desktop.less

```diff
@@ -229,13 +230,16 @@ gmf-layertree ul .treenode {
     position: relative;
 
     > .btn + .btn {
-      margin-top: 0;
+      margin-top: -1px;
     }
 
     .btn {
       background-color: @brand-primary;
-      border: 0;
+      margin-left: 0;
+      border-right-width: 0;
+      border-left-width: 0;
       border-radius: 0 !important;
```

Comment:
I'd prefer to keep this. This will prevent weird behaviour if a user chooses to force the radiuses back to a value greater than 0.